docs/Design/Alpha/Data Model.md

```diff
@@ -0,0 +1,215 @@
+by each entity and how the entities relate to each other.
+
+```mermaid
+---
+title: Initial Data Model
+---
+
+erDiagram
+    artist {
+        int id PK 
+        string(31) slug UK "Unique human-readable URL-safe identifier"
+        string(255) name "The name of the Artist"
+        string(16384) description "Long text description (Nullable)"
+        string(63) location "The city in which the Artist is based (Nullable)"
+        boolean is_group "Indicates if this Artist is a group or individual"
+        string(255) profile_image_uri 
+        string created_at "Timestamp"
+        string updated_at "Timestamp"
+    }
+    album {
+        int id PK 
+        string(31) slug UK "Unique human-readable URL-safe identifier"
+        string(255) name "The name of the Album"
+        string(16384) description "Long text description (Nullable)"
+        string(enum) type "Full-length, EP, Single, etc."
+        int price_cents "USD cent value for price to buy full Album"
+        string cover_art_uri 
+        string release_date "The release date of the Album (Nullable timestamp)"
+        int release_year "The release year of the Album (Nullable)"
+        string created_at "Timestamp"
+        string updated_at "Timestamp"
+    }
+    track {
+        int id PK 
+        string(31) slug UK "Unique human-readable URL-safe identifier"
+        string(255) name "The name of the Track"
+        string(16384) description "Long text description (Nullable)"
+        int track_number "The number which describes the Track's ordering within its Album"
+        int price_cents "USD cent value for price to buy individual Track (Nullable)"
+        boolean is_streamable "Allows Users to stream the Track for free"
+        boolean is_previewable "Allows Users to preview the Track before purchase"
+        int streams "The number of streams the Track has gotten"
+        int downloads "The number of downloads the Track has gotten"
+        string content_uri "URI from which to stream or download the Track"
+        int runtime_seconds "The length of the Track in seconds"
+        string content_type "The format (MIME-type) of the Track's audio file"
+        int size_bytes "The size in bytes of the Track's audio file"
+        int sample_rate "The sample rate of the Track's audio file"
+        int bit_depth "The bits per sample of the Track's audio file"
+        string(255) checksum_sha256 "The SHA-256 checksum of the Track's audio file"
+        string created_at "Timestamp"
+        string updated_at "Timestamp"
+    }
+    playlist {
+        int id PK
+        string(31) slug UK "Unique human-readable URL-safe identifier"
+        string(255) name "The name of the Playlist"
+        string(16384) description "Long text description (Nullable)"
+        string(255) cover_art_uri "Nullable"
+        int owner_id FK "User who owns this Playlist"
+        string created_at "Timestamp"
+        string updated_at "Timestamp"
+    }
```

Comment:
I think for what we're trying to achieve currently, playlists should not be part of the MVP, and should def be something later down the line.

Reply:
Removed from MVP